recipe/bld.bat

```diff
@@ -17,3 +17,4 @@ move share %LIBRARY_PREFIX%\
 if errorlevel 1 exit 1
 
 cp %RECIPE_DIR%\Windows-Flang-Fortran.cmake %LIBRARY_PREFIX%\share\cmake-3.10\Modules\Platform\
+cp %RECIPE_DIR%\FindBoost.cmake %LIBRARY_PREFIX%\share\cmake-3.10\Modules\
```

Comment:
Is both patching and copying necessary?

Reply:
We could copy or patch for both source and binary releases. I started with the patch the realized the windows package is binary-only so I just copied the patched module for windows.

Reply:
Ah, okay